algobot/interface/configuration.py

```diff
@@ -73,7 +73,7 @@ def __init__(self, parent: QMainWindow, logger: Logger = None):
         }
 
         self.lossTypes = ("Trailing", "Stop")
-        self.takeProfitTypes = ('Stop',)
+        self.takeOrderTypes = ('Stop',)
```

Comment:
This should be the same, because notice we have loss types above

Reply:
Ouch this was a bad reactor, let me fix it 🎇